config/internal/apiserver/default/deployment.yaml.tmpl

```
@@ -206,19 +206,17 @@ spec:
             - mountPath: /etc/tls/private
               name: proxy-tls
           {{ end }}
-          {{ if or .APIServer.EnableSamplePipeline .CustomCABundle }}
-            {{ if .APIServer.EnableSamplePipeline }}
+          {{ if or .APIServer.EnableSamplePipeline .APIServer.EnableInstructLabPipeline}}
             - name: sample-config
               mountPath: /config/sample_config.json
               subPath: sample_config.json
             - name: sample-pipeline
               mountPath: /samples/
-            {{ end }}
-            {{ if .CustomCABundle }}
+          {{ end }}
+          {{ if .CustomCABundle }}
             - mountPath: {{ .CustomCABundleRootMountPath  }}
               name: ca-bundle
-            {{ end }}
-          {{ end }}
+          {{ end }}          
```

This should fix the CI failure.
```suggestion
          {{ end }}
```